libsolidity/formal/SMTEncoder.cpp

```diff
@@ -1111,11 +1111,11 @@ void SMTEncoder::arrayPop(FunctionCall const& _funCall)
 	symbArray->increaseIndex();
 	m_context.addAssertion(symbArray->elements() == oldElements);
 	auto newLength = smtutil::Expression::ite(
```

Comment:
Was this unused before?

Reply:
Actually yes... also only noticed in this PR